src/main/java/de/zalando/aruha/nakadi/repository/EventTypeRepository.java

```diff
@@ -6,13 +6,22 @@
 import de.zalando.aruha.nakadi.exceptions.NoSuchEventTypeException;
 
 import java.util.List;
+import java.util.Optional;
 
 public interface EventTypeRepository {
 
     void saveEventType(EventType eventType) throws InternalNakadiException, DuplicatedEventTypeNameException;
 
     EventType findByName(String name) throws InternalNakadiException, NoSuchEventTypeException;
 
+    default Optional<EventType> findByNameO(String name) throws InternalNakadiException {
```

Comment:
I don't really like that we now have two methods in this repository that do exactly the same but have different notations.   I can imagine that someone will create a third method that returns null because he doesn't like optionals 😄